iceoryx_dds/examples/docker/publisher_node.entrypoint

```diff
@@ -14,6 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GATEWAY_CONF_PATH=/etc/iceoryx/gateway_config.toml
+
+mkdir -p /etc/iceoryx
+
+if [[ ! -f "$GATEWAY_CONF_PATH" ]]; then
+    cat <<EOF >> /etc/iceoryx/gateway_config.toml
```

Comment:
This config is supposed to be generic for all gateways.

Reply:
If it is generic, why does it live in `iceoryx_dds` then?

Reply:
@mossmaurice, I think ithier wants that every gateway uses that config and this is just an example which uses the config

How about `/etc/iceoryx/gateway_config.d/generic.toml (or fallback.toml)` and if a gateway wants to have a custom config, it would put into e.g. `/etc/iceoryx/gateway_config.d/dds.toml`?

For me it's not yet fully clear how it is supposed to be used.